package.json

```diff
@@ -8,12 +8,7 @@
     "build": "rm -rf ./built && npx tsc && chmod +x ./built -R"
   },
   "bin": {
-    "caprover": "./built/commands/caprover.js",
-    "caprover-deploy": "./built/commands/deploy.js",
-    "caprover-list": "./built/commands/list.js",
-    "caprover-login": "./built/commands/login.js",
-    "caprover-logout": "./built/commands/logout.js",
-    "caprover-serversetup": "./built/commands/serversetup.js"
+    "caprover": "./built/commands/caprover.js"
```

Comment:
The only entry point is caprover cli; sub commands need to be built by the main cli and they can't be called directly. I don't think it could be a problem.

Reply:
That's what I expected as well, but it turned out that, if installed using yarn, it becomes a problem. See this:
https://github.com/caprover/caprover/issues/338#issuecomment-450513064